packages/scanner-global-library/README.md

```diff
@@ -0,0 +1,6 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+The `scanner-global-library` package contains the common web page scanning logic used by both service and command line packages.
```

Comment:
I don't see how it's used in the service package

Reply:
Right, I'll add this package to the service in next PR